cwms-servers/CWMS_HOME/config/system_config/cwms_system_logging.properties

```diff
@@ -11,17 +11,16 @@
 #  (all loggers are children of the root logger)
 #  Create handlers
 
-# handlers = java.util.logging.ConsoleHandler,com.splunk.logging.HttpEventCollectorLoggingHandler,biz.paluch.logging.gelf.jul.GelfLogHandler
 handlers = java.util.logging.ConsoleHandler,biz.paluch.logging.gelf.jul.GelfLogHandler
 
 #  Set the default logging level for the root logger
 
-.level = INFO
+.level = FINE	
```

Comment:
I'd recommend against this as INFO is the default for all CWMS, target the specifics you want at FINE if you need this.

Reply:
updated

Reply:
Added ticket for updating GELF log handler to prevent stack overflow: https://github.com/USACE-WaterManagement/cwms-logging/issues/4